### PR TITLE
Relay fixes and presence call bugs

### PR DESCRIPTION
**Pull request by ruth0048**

### CODE REVIEW

**Comment by ruth0048** [comment]
tested with java relay app and unit tests

File: brainCloud/main/java/com/bitheads/braincloud/services/PresenceService.java [hunk 1]
```diff
@@ -91,7 +91,7 @@ public void getPresenceOfUsers(ArrayList<String> profileIds, boolean includeOffl
 	{
 		try {
 			JSONObject data = new JSONObject();
-			data.put(Parameter.profileIds.name(), profileIds);
+			data.put(Parameter.profileIds.name(), new JSONArray(profileIds));
```
Comment (davidstl) [inline brainCloud/main/java/com/bitheads/braincloud/services/PresenceService.java:94]: Sup with that? Is this related to Relay Changes ?

Comment (ruth0048) [reply brainCloud/main/java/com/bitheads/braincloud/services/PresenceService.java:94]: this was part of the Jira. https://playbrains.atlassian.net/browse/BCLD-5914

Comment (davidstl) [reply brainCloud/main/java/com/bitheads/braincloud/services/PresenceService.java:94]: oh right

**Review by davidstl — APPROVED** [review]
Ya I guess that's good if you tested all the things
